debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/J9ObjectFieldOffsetIterator_V1.java

```diff
@@ -214,7 +215,20 @@ private void fieldOffsetsFindNext() throws CorruptDataException {
 							J9ClassPointer fieldClass = valueTypeHelper.findJ9ClassInFlattenedClassCacheWithFieldName(instanceClass, J9ROMFieldShapeHelper.getName(localField));
 							if (valueTypeHelper.isJ9FieldIsFlattened(fieldClass, localField)) {
 								UDATA size = null;
-								if (valueTypeHelper.isJ9ClassLargestAlignmentConstraintDouble(fieldClass)) {
+								boolean forceDoudbleAlignment = false;
```

Comment:
Please fix the name ('Doudble' -> 'Double') and don't include redundant initialization. The compiler will issue an error if it can't prove that a variable is always assigned a value before it is used.

Reply:
Fixed.